app/common/clinical-services/components/formField.js

```diff
@@ -151,7 +151,7 @@
 
           if (angular.isDefined(model.value)) {
 
-            if (angular.isDefined(model.value[answer.uuid])) {
+            if (angular.isDefined(model.value[answer.uuid]) && model.value[answer.uuid] != 'undefined') {
```

Comment:
It's working now, but there is this rule [here](https://eslint.org/docs/rules/eqeqeq) that i think we should be enforcing.

Reply:
Fixed, please review again.